recipes/tassel/3.0.137/build.sh

```diff
@@ -0,0 +1,31 @@
+BINARY_HOME=$PREFIX/bin
+TASSEL_HOME=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
+
+# Download TASSEL4 from its public repo
+git clone git://git.code.sf.net/p/tassel/tassel3-standalone tassel3-standalone
+cd tassel3-standalone
+git checkout V$PKG_VERSION
+
+# Copy source to the conda environment
+mkdir -p $TASSEL_HOME
+cp -R * $TASSEL_HOME/
+
+# Fix shabang lines
+sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $TASSEL_HOME/*.pl
+
+# Create wrapper script in bin to call TASSEL's launch scripts
+cat >"${BINARY_HOME}/run_pipeline.sh" <<EOF
+exec $CONDA_PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/run_pipeline.pl "\$@"
+EOF
+
+cat >"${BINARY_HOME}/run_anything.sh" <<EOF
+exec $CONDA_PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/run_anything.pl "\$@"
+EOF
+
+cat >"${BINARY_HOME}/start_tassel.sh" <<EOF
+exec $CONDA_PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM/start_tassel.pl "\$@"
```

Comment:
We shouldn't rely on `$CONDA_PREFIX` while building, but use `$PREFIX` instead. Since those variables are expanded at build time, you can just use `$TASSEL_HOME` instead of `$CONDA_PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM` in all three wrapper scripts.

Reply:
Removed any reference to ```CONDA_PREFIX``` in the latest commit.